tests/AppBundle/Availability/AvailabilityFunctionalTest.php

```diff
@@ -258,7 +258,7 @@ public function teamLeaderUrlProvider()
             array('/kontrollpanel/hovedstyret/oppdater'),
 
             array('/kontrollpanel/opptakadmin/teaminteresse'),
-            array('/kontrollpanel/opptakadmin/teaminteresse?department=2&semester=1'),
+            array('/kontrollpanel/opptakadmin/teaminteresse?department=1&semester=1'),
```

Comment:
Here is one of the tests that had the id changed. I suspect this was a typo and was supposed to reference department 1, as the TeamLeader client in question was also a member of department 1.

Reply:
⚠️ THIS SHOLD BE APPROVED BEFORE MERGE ⚠️

Reply:
Here you can see the typo I think has occurred. In the top line, the semester with id = 4 is changed to `department=1&semester=1`. I the bottom line, the same semester is changed to `department=2&semester=1`, which causes the test to fail. 
![Screenshot 2020-09-24 at 12 13 52](https://user-images.githubusercontent.com/31702423/94132383-82524780-fe5f-11ea-8a6e-67805325b360.png)

The change came with `Fjern department fra semester`, #1048 by @sigtot, so perhaps he has some insight into this 😄

Reply:
Lgtm